lib/editor/components/timetable/TripSeriesModal.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+// $FlowFixMe: Flow doesn't know about useState: https://stackoverflow.com/questions/53105954/cannot-import-usestate-because-there-is-no-usestate-export-in-react-flow-with
+import React, {useState} from 'react'
+import {Button, Modal} from 'react-bootstrap'
+
+import * as tripActions from '../../actions/trip'
+import {getComponentMessages} from '../../../common/util/config'
+import HourMinuteInput from '../HourMinuteInput'
+
+// TODO: fill in "any" prop types
+type Props = {
+  addNewTrip: typeof tripActions.addNewTrip,
+  constructNewRow: any,
+  onClose: any,
+  show: boolean
+}
+
+const TripSeriesModal = (props: Props) => {
+  const [startTime, setStartTime] = useState(0)
+  const [headway, setHeadway] = useState(0)
+  const [endTime, setEndTime] = useState(0)
+  const [show, setShow] = useState(false)
+
+  const messages = getComponentMessages('TripSeriesModal')
+
+  const onClose = () => {
+    setShow(false)
```

Comment:
Great point. Removed that since it's unnecessary in https://github.com/ibi-group/datatools-ui/pull/892/commits/7bc484d5276ea6e8684d35a5f6b864e391d87111